examples/realTimeScene.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python3
+
+import Sofa.Core
+import Sofa.Simulation
+import SofaRuntime
+import os
+import Sofa.splib.animation
+from time import time
+SofaRuntime.importPlugin("SofaOpenglVisual")
+
+_runAsPythonScript = False
```

Comment:
Is this really needed ? I don't see any use of this one int he createScene function.

Reply:
Ok i filly found where it is used....but still don't see the point, the simulation should be the same if it is exectued in runSofa or un a python3 environment isn't it ?

Reply:
This example was initially based on the ControllerScene.py example, however it appears it is no longer needed in my scene, and I forgot about it. Thanks, I will delete it.